Dockerfile

```diff
@@ -6,7 +6,8 @@ RUN mkdir /app
 WORKDIR /app
 
 RUN curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.13.1/maxwell-0.13.1.tar.gz | tar zxvf -
-RUN mv maxwell-*/* .
+RUN cp -r maxwell-*/* .
+RUN rm -rf maxwell-*
```

Comment:
It's better to do it in the same step, something like, 
```
RUN cp -r maxwell-*/* . && \
rm -rf maxwell-*
```
Doing in two steps means, the maxwell tar ball is left in the previous layer which from the rm i assume you are trying to not leave around.